sklearn/cluster/k_means_.py

```diff
@@ -875,6 +875,7 @@ def fit(self, X, y=None):
         Parameters
         ----------
         X : array-like or sparse matrix, shape=(n_samples, n_features)
+            Coordinates of training data points to cluster
```

Comment:
@NelleV  Correctly stated. I made a mistake there.  How about extending 'sparse matrix with each element representing the feature vector for the training data points to cluster'?